src/main/kotlin/app/revanced/patcher/Context.kt

```diff
@@ -0,0 +1,114 @@
+package app.revanced.patcher
+
+import app.revanced.patcher.apk.Apk
+import app.revanced.patcher.apk.ApkBundle
+import app.revanced.patcher.apk.ResourceFile
+import app.revanced.patcher.util.method.MethodWalker
+import org.jf.dexlib2.iface.Method
+import org.w3c.dom.Document
+import java.io.Closeable
+import java.io.InputStream
+import java.io.OutputStream
+import javax.xml.parsers.DocumentBuilderFactory
+import javax.xml.transform.TransformerFactory
+import javax.xml.transform.dom.DOMSource
+import javax.xml.transform.stream.StreamResult
+
+/**
+ * A common class to constrain [Context] to [BytecodeContext] and [ResourceContext].
+ * @param apkBundle the [ApkBundle] for this context.
```

Comment:
Please capitalize the first letter in descriptions
```suggestion
 * @param apkBundle The [ApkBundle] for this context.
```